src/main/java/org/engine/process/performance/EnginePerformanceSingle.java

```diff
@@ -48,7 +47,7 @@
  *
  */
 
-public class EnginePerformanceFromBeginning extends InnerService {
+public class EnginePerformanceSingle extends InnerService {
```

Comment:
change EnginePerformanceSingle to EnginePerformanceSingleMessage and add some comments to explain that this is the initial implementation, just for reference, and we are not using it anymore